pkg/nfd-master/nfd-master.go

```diff
@@ -249,6 +239,18 @@ func (m *nfdMaster) Run() error {
 			if err := m.configure(m.configFilePath, m.args.Options); err != nil {
 				return err
 			}
+
+			// restart NFD API controller
+			if m.nfdController != nil {
+				klog.Info("stopping the nfd controller")
```

Comment:
nit: symmetry with the start message

```suggestion
				klog.Info("stopping the nfd api controller")
```